js/ui/select_box.js

```diff
@@ -384,15 +384,13 @@ var SelectBox = DropDownList.inherit({
     },
 
     _renderInputValue: function() {
-        this._renderInputValueAsync = function() {
-            this._renderTooltip();
-            this._renderInputValueImpl();
-            this._refreshSelected();
-        };
+        return this.callBase().always(this._renderInputValueAsync.bind(this));
+    },
 
-        return this.callBase().always((function() {
-            this._renderInputValueAsync();
-        }).bind(this));
+    _renderInputValueAsync: function() {
```

Comment:
@mrjohn0011 

I reverted the `_dispose` method implementation in [this commit](https://github.com/DevExpress/DevExtreme/pull/737/commits/f47780b633d41a418a2e69cd4c6525f8a987c9e4)